Index: core_functions.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- core_functions.py	(revision afbac1ce1ca2a1e771ff4941bb8178988b9ba0dd)
+++ core_functions.py	(date 1590525184993)
@@ -1,6 +1,43 @@
 import csv
 import datetime
 
+# cof_id,name,min_rd_count,max_distance,field of fire,number_of_targets,target_001_type,target_001_positions,target_002_type,target_002_positions,starting_position,firing positions,starting conditions,reloads,transition,Movement,Judgement,failure simulations,scoring,instructions
+cof_selected = {'cof_id': 1,
+                'name': 'F.A.S.T Drill',
+                'min_rd_count': 14,
+                'max_distance': '7 yards',
+                'field of fire': 180,
+                'number_of_targets': 2,
+                'target_001_type': '3x5',
+                'target_001_positions': '360_252_70_180_0',
+                'target_002_type': 'circle 8in',
+                'target_002_positions': '360_252_60_180_0',
+                'starting_position': 'holstered',
+                'firing positions': 1,
+                'starting conditions': 'loaded_chambered_Y_1',
+                'reloads': 1,
+                'transition': 0,
+                'Movement': 0,
+                'Judgement': 0,
+                'failure simulations': 0,
+                'scoring': 1,
+                'instructions': "You will start with your weapon holstered and chambered, with 1(one) round in the magizine. You will have a second magizine loaded with 4 (four) rounds in a carrier. On the command to fire you will 1. Draw from concealment, 2. fire 2 rounds on target 1, 3. reload - slide-lock, 4. fire 4 on target 2. Then make clear and holster your weapon"}
+
+
+def break_target_location(target_name_num, value):
+    target_name_num = value.split("_")
+    degree_from_zero = target_name_num[0]
+    # print(f'Target is at {degree_from_zero} degrees from shooter ')
+    distance_from_zero = target_name_num[1]
+    # print(f'{distance_from_zero} inches away,')
+    hight_above_zero = target_name_num[2]
+    # print(f'The top of the target is {hight_above_zero} above the ground')
+    facing_angle = target_name_num[3]
+    # print(f'facing {facing_angle} degrees')
+    angle_of_rotation = target_name_num[4]
+
+
+break_target_location('target_round_001', '360_000_70_180_0')
 
 # Splash screen, user is show
 # # a filterable list of history combined skills results
@@ -89,6 +126,7 @@
         cof_scoring = selected_cof_inprog[17]
         cof_instructions = selected_cof_inprog[18]
 
+        # break_target_location('target_location001', cof_target_001_positions)
         target_location001 = cof_target_001_positions.split("_")
         degree_from_zero = target_location001[0]
         distance_from_zero = target_location001[1]
@@ -136,13 +174,10 @@
     f.close()
 
 
-# with open("user_local_hist_cof.csv", mode='r')as loc_hist:
-#     csv_reader = csv.DictReader(loc_hist)
-#     for row in csv_reader:
-#         print(row)
-#         shotnumber = int(row['shot_number'])
-#         print(shotnumber)
-#         print(shotnumber + 1)
+with open("user_local_hist_cof.csv", mode='r')as loc_hist:
+    csv_reader = csv.DictReader(loc_hist)
+    for row in csv_reader:
+        print(row)
+        shotnumber = int(row['shot_number'])
 
 
-record_results()
Index: user_local_hist_cof.csv
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- user_local_hist_cof.csv	(revision afbac1ce1ca2a1e771ff4941bb8178988b9ba0dd)
+++ user_local_hist_cof.csv	(date 1590523985988)
@@ -1,5 +1,7 @@
-date,shot_number,Hit,score,time,target,hit location,reload
-2020-05-22 21:13:06.035995, 1, y, 3, 4.65,8in, 4x4, n
-2020-05-22 21:43:00.237648, 2, Y, 4, 3.22,8in, 4x3, N
-2020-05-22 21:44:06.877975, 3, Y, 0, 5.2,4in, 8x8, n
-2020-05-22 21:46:05.028365, 4, Y, 2, 3.2,3in, 3x3, n
\ No newline at end of file
+date,shot_number,Hit,score,time,target,hit location,reload, cof_selected
+2020-05-22 21:13:06.035995, 1, y, 3, 1.00,3x5, 2x4, n, 1
+2020-05-22 21:43:00.237648, 2, Y, 4, 3.22,3x5, 1x4, N, 1
+2020-05-22 21:44:06.877975, 3, Y, 0, 15.2,circle 8in, 4x5, y, 1
+2020-05-22 21:46:05.028365, 4, Y, 2, 16.2,circle 8in, 3x3, n, 1
+2020-05-22 21:46:05.028365, 5, Y, 2, 17.2,circle 8in, 3x4, n, 1
+2020-05-22 21:46:05.028365, 6, Y, 2, 18.2,circle 8in, 4x3, n, 1
\ No newline at end of file
